CMakeLists.txt

```diff
@@ -65,8 +65,8 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${OPAE_SDK_SOURCE}/cmake/modules")
 ############################################################################
 set(OPAE_PYTHON_VERSION 2.7 CACHE STRING "Python version to use for building/distributing pyopae")
 set_property(CACHE OPAE_PYTHON_VERSION PROPERTY STRINGS 2.7 3.6 3.5 3.4 3.3)
-find_package(PythonLibs ${OPAE_PYTHON_VERSION})
 find_package(PythonInterp ${OPAE_PYTHON_VERSION})
+find_package(PythonLibs ${OPAE_PYTHON_VERSION})
```

Comment:
Is there a difference for switching this two lines?

Reply:
Yes, this way you will find the development libraries (and headers) that come packaged with the Python interpreter found.
In my case, I'm using a custom installation of Python 2.7.15 in a non-standard location.
Because of this, I can't find the cmake package until _after_ I've found the interpreter.